legos/memes.py

```diff
@@ -168,6 +182,18 @@ def _construct_url(self, meme):
     def get_name(self):
         return 'memes'
 
-    def get_help(self):
-        return ('Create memes through natural text. See https://github.com/'
+    def get_help(self, **kwargs):
+        subcommands = {
+                'keywords': '''In addition to natural language, lego.memes \
+                            supports keywords to create memes. To use this \
+                            feature, use this pattern: \
+                            `keyword: top line, bottom line`\n'''
+                            'Try `!help memes list` for a list of keywords',
+                'list': ', '.join([*self.templates])
+                }
+
+        if ('sub' in kwargs):
+            return subcommands[kwargs['sub']]
+        else:
+            return ('Create memes through natural text. See https://github.com/'
```

Comment:
This needs some serious formatting love. There's no whitespace between sentences and no line breaks. Make it look nice plz?

Reply:
the new version of the subcommands look good.